<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Error_Dialog</class>
 <widget class="QDialog" name="Error_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>470</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Error_Dialog">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>433</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Error_Dialog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BitRock">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GPL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_perl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Qt_logo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DOMINO">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
